controller/transformer.py

```diff
@@ -0,0 +1,18 @@
+class Transformer:
+    def __init__(self, cid):
+        self._cid = cid
+
+    def transform_startapp_data(self, service):
+        return_data = {}
+        for i in service.data:
+            if i['campaignId'] == self._cid:
+                return_data.update({"impressions": i["impressions"], "clicks": i["clicks"], "spent": i["spent"]})
+        return return_data

```

Comment:
return_data must be an object which has several behaviors *GetImpressions()* or something